[Bug]: implicit dependency on lyne-design-tokens

### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Functionality

### It affects the following packages

Components

### Which version of Lyne Components are you using

3.2.0

### Operating system

win 10

### Browser / Browser version

n/a

### Input mode

None

### Additional settings

_No response_

### What happened?

When upgrading to the latest lyne-angular@20.0.0-rc.1 and lyne-elements@3.2.0 there occurs the following compile error:

```
X [ERROR] Can't find stylesheet to import.
  ╷
7 │ @use '@sbb-esta/lyne-design-tokens/dist/scss/sbb-variables.scss' as sbb-tokens;
  │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules\@sbb-esta\lyne-elements\core\styles\core\mediaqueries.scss 7:1  @forward
  node_modules\@sbb-esta\lyne-elements\core\styles\_index.scss 2:1             @forward
  node_modules\@sbb-esta\lyne-elements\_index.scss 1:1                         @use
  src\variables.scss 4:1                                                       @use
  src\app\styles\_layout.scss 1:1                                              @use
  src\styles.scss 3:1                                                          root stylesheet [plugin angular-sass]

    angular:styles/global:styles:2:8:
      2 │ @import 'src/styles.scss';        
```

### Steps to reproduce the issue

1. Use lyne-angular + lyne-elements
2. Import lyne-elements SCSS: `@use '@sbb-esta/lyne-elements' as styles;`
3. Observe the error


### Relevant log output

```shell

```

--- a/tools/vite/copy-sass.ts
+++ b/tools/vite/copy-sass.ts
@@ -22,8 +22,9 @@ export function copySass(sassRoot: string): PluginOption {
           type: 'asset',
           fileName: file,
           source: readFileSync(join(viteConfig.root, file), 'utf8').replace(
-            /(@use|@import) '(node_modules\/[^']+)'/g,
+            /(@use|@import) '((?!\.|sass:)[^']+)'/g,
             (_m, useOrImport, useFile: string) => {
+              useFile = join('node_modules', useFile);
               const outFile = join(sassRoot, useFile.replaceAll('/', '_'));
               this.emitFile({
                 type: 'asset',

